src/socket.c

```diff
@@ -125,6 +125,18 @@ static int connSocketConnect(connection *conn, const char *addr, int port, const
  * move here as we implement additional connection types.
  */
 
+static void connSocketShutdown(connection *conn, int shut_rd, int shut_wr) {
+    if (conn->fd == -1) return;
+
+    if (shut_rd && shut_wr) {
+        shutdown(conn->fd, SHUT_RDWR);
+    } else if (shut_rd) {
+        shutdown(conn->fd, SHUT_RD);
+    } else if (shut_wr) {
+        shutdown(conn->fd, SHUT_WR);
+    }
+}
```

Comment:
i think we can drop the shut_wd and shut_wr arguments and always pass RDWR.
@yossigo WDYT?

Reply:
yes, i am for it, now we don't have other cases that need these flags